### PR TITLE
Added ConsolidateArtifacts.bat

### PR DESCRIPTION
**Pull request by AntoineConffx**
Added ConsolidateArtifacts.bat that extracts the json.hpp from src folder and moves it to artifacts folder (will be created if doesn't exist)

### CODE REVIEW

File: ConsolidateArtifacts.bat [hunk 1]
```diff
@@ -0,0 +1,27 @@
+setlocal EnableDelayedExpansion
+
+:: Libraries from local folder
+@SETLOCAL
+@ECHO off
+call "%VS140COMNTOOLS%..\..\VC\bin\vcvars32.bat"
```
Comment (mbarnes-sb) [inline ConsolidateArtifacts.bat]: This is 32 bit env. Don't you need 64 bit builds?

File: ConsolidateArtifacts.bat [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+set "CURR_DIR=."
+
+::remove original artifacts (if any)
+if exist "OUT_DIR" rmdir "%OUT_DIR%" /s /q
+::create artifacts folder
+mkdir "%OUT_DIR%"
+
+::go to json11 folder and move json.hpp into artifact folder
+xcopy  "src\*.hpp" "%OUT_DIR%" /y/s/q/i
+if !errorlevel! neq 0 exit /b !errorlevel!
```
Comment (mbarnes-sb) [inline ConsolidateArtifacts.bat]: Modern syntax for this statement is `if errorlevel 1 exit /b %errorlevel%`

**Review by mbarnes-sb — CHANGES_REQUESTED** [review]
please review comments and provide feedback.

File: ConsolidateArtifacts.bat (hunk 2 above)
Comment (AntoineConffx) [reply ConsolidateArtifacts.bat]: I have updated that file with another small bug fix on line 12.

File: ConsolidateArtifacts.bat (hunk 1 above)
Comment (AntoineConffx) [reply ConsolidateArtifacts.bat]: I am now using 64-bit visual studio compiler but the blosc project doesn't provide any configuration properties for building in 64-bit, it's only provided for Win32.
Also added clean up of temporary files and folders used for building

**Review by mbarnes-sb — APPROVED** [review]
looks okay

File: ConsolidateArtifacts.bat [hunk 3]
```diff
@@ -0,0 +1,27 @@
+setlocal EnableDelayedExpansion
+
+:: Libraries from local folder
+@SETLOCAL
+@ECHO off
+
+
+set "OUT_DIR=./Artifacts"
```
Comment (krunal-patel-fl4re) [inline ConsolidateArtifacts.bat]: should be 'artifacts' with small 'a'

**Pull request merged**